tests/compas_timber/test_joint.py

```diff
@@ -30,7 +29,7 @@ def example_beams():
     beams = []
     for index, line in enumerate(centerlines):
         b = Beam.from_centerline(line, w, h)
-        b.key = index
+        b.graph_node = index
```

Comment:
I agree. but honestly the more I think of it calling it `key` didn't quite make sense..

Reply:
it's hard to tell because I was used to it for a long time. generally some of the `Graph` implementation is mysterious to me. like I'd expect `graph_node` to return a `GraphNode` object (which doesn't exist, `Graph` is mostly a dict..)

I think the right course of action is to leave these internal workings of `Graph` internal. All references to elements that are not instances now go with the `guid`. I'm using `graph_node` somewhere in the unittests just because it was easier than to use the guid.